icu4c/source/i18n/measunit.cpp

```diff
@@ -537,9 +537,9 @@ static const char * const gSubTypes[] = {
     "solar-mass",
     "stone",
     "ton",
-    "one",
-    "percent",
-    "permille",
+    "", // FIXME(review): this should be generated by Java, from resource files.
+    "percent", // FIXME(review): should this be here? both percent and permille
```

Comment:
For both of these items, file a bug to follow up in ICU 68, and make the TODO point to that bug.

Reply:
Did the "one" that you added come from ICU4J as per the normal instructions? Or was it a hand-edit?

I'm wondering if it matters that I'm hand-editing this generated code - I feel I'm asking for trouble, but diffs and unit tests should at least catch that trouble?